scripts/deploy/config/databend-query-node-2.toml

```diff
@@ -39,3 +23,26 @@ clickhouse_handler_host = "0.0.0.0"
 clickhouse_handler_port = 9002
 
 namespace = "test_cluster"
+
+[log]
+log_level = "ERROR"
+log_dir = "./_logs"
+
+# Meta service.
+[meta]
+meta_address = "0.0.0.0:9191"
+meta_username = "root"
+meta_password = "root"
+
+# Storage config.
+[storage]
+storage_type = "disk|dfs|s3"
```

Comment:
not sure whether persistence storage already implemented in stateless CI pipeline, if we  have not incorporate storage in test, consider put possible values into annotations and leave config field empty